orttraining/orttraining/core/framework/gradient_graph_builder.cc

```diff
@@ -32,7 +32,6 @@ GradientGraphBuilder::GradientGraphBuilder(Graph* graph,
   auto rule_based_graph_transformer =
       onnxruntime::make_unique<RuleBasedGraphTransformer>("pre_training_rule_based_graph_transformer");
   rule_based_graph_transformer->Register(make_unique<InsertMaxPoolOutput>());
-  rule_based_graph_transformer->Register(make_unique<AdjustBatchNormOutputs>());
 
   graph_transformation_mgr_.Register(std::move(rule_based_graph_transformer),
```

Comment:
InsertMaxPoolOutput can also be removed here. 
actually, graph_transformation_mgr_ can be removed from this class.

Reply:
Done.